src/vireshJJun20/Assignment27/MeetingRoom.java

```diff
@@ -0,0 +1,113 @@
+/*
+Program to demonstrate overloading behavior.
+
+Create a class MeetingRoom having method bookMeetingRoom which should provide
+below facility:
+meeting room names: Nalanda, Takshashila, AgraFort, and PratapGadh
+
+a. If any user call method bookMeetingRoom() at that time�Nalanda� meeting room will be booked for 1 hr.
+b. Write a method that helps to book a meeting room other than "Nalanda" for default timing would be 1 hr.
+c. Write a method which helps to book a meeting room by giving the name of the meeting room and given hrs.
+d. displayInfo() method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs].
+e. displayAvilableMeetingRoomForBooking() method will print available meeting room names. 
+     [i.e. If Nalanda and Takshashila is booked then this method should print "Available meeting rooms: AgraFort and 
+     PratapGadh"]
+Note: Using enum
+*/
+package vireshJJun20.Assignment27;
+
+public class MeetingRoom {
+
+	static int nalandaCtr, takshashilaCtr, agraFortCtr, pratapGadhCtr, nalandahours, takshashilahours, agraforthours, pratapgadhahours;
```

Comment:
Flag is more suitable then use int for variables like nalandaCtr, takshashilaCtr, agraFortCtr, pratapGadhCtr.
This is debatable topic in the interview. Never do this again.